src/main/java/practice/pankaj/CountOccurance.java

```diff
@@ -0,0 +1,35 @@
+package practice.pankaj;
+import java.util.Scanner;
+
+public class CountOccurance {
+
+
+
+        public static void main(String args[])
+        {
+            String strn;
+            int i, len;
+            int counter[] = new int[256];
+            Scanner scan = new Scanner(System.in);
+            System.out.print("Please enter a string: ");
+            strn = scan.nextLine();
+
+            len = strn.length();
+
+            for (i = 0; i < len; i++)
+            {
+                counter[(int) strn.charAt(i)]++;
+            }
+
+            for (i = 0; i < 256; i++)
+            {
+                if (counter[i] != 0)
+                {
+
+                    System.out.println((char) i + " --> " + counter[i]);
+                }
+            }
+        }
+
```

Comment:
public static void main (String[]args)
    {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter long sentence ");
        String sentence = sc.nextLine();
        int count=1;
        Map<String,Integer> words = new HashMap<>();
        String[] word = sentence.split(" ");
        for(String w : word)
        {
            if(words.containsKey(w))
            {
                words.put(w,count+1);
            }
            else
            {
                words.put(w,count);
            }
        }
        System.out.println(words);
    }